File: Linkedlist.py
class Node:
	def __init__(self,data=None):
		self.data=data
		self.next=None




class LinkedList:
	def __init__(self):
		self.head = Node()

	def insertnode(self,data):
		newnode = Node(data)
		curr = self.head
		#while curr.next!=None:
			#curr = curr.next
		curr.next = newnode


			


test = LinkedList()
test.insertnode(1)
print(test.head.__dict__)
		

File: LinkedList.js
class Node{
    constructor(data=null){
        this.data=data;
        this.next=null
    }
}

class Linkedlist{
    constructor(){
        this.head = null;
        this.size=0;
    }

    insertdata(data){
        let newnode = new Node(data);
        let currentitem;
        if(this.head == null){
            this.head=newnode;
        }else{
            currentitem = this.head;
            while(currentitem.next){
                currentitem = currentitem.next;
            }
            currentitem.next = newnode;
        }
        
        //console.log(currentitem);
       
        //currentitem.next = newnode

    }
}

let list = new Linkedlist()
list.insertdata(2)
console.log(list)
